### PR TITLE
Iconik port

### PR DESCRIPTION
**Pull request by sonicaj**
This PR introduces port for iconik

### CODE REVIEW

File: net/iconik/distinfo [hunk 1]
```diff
@@ -0,0 +1,3 @@
+TIMESTAMP = 1567714449
+SHA256 (iconik_storage_gateway-latest.txz) = 6b241a5d39d794a2b94e9743728614349798d0d65efa21a237c89395c0c40c79
```
Comment (william-gr) [inline net/iconik/distinfo]: Lets try to find out if they can provide a link for a static file, that will not change over time (e.g. no latest)

File: GIDs [hunk 2]
```diff
@@ -205,7 +205,7 @@ meta1q:*:261:
 meta1c:*:262:meta1s
 meta1m:*:263:meta1s,meta1q
 meta1:*:264:
-# free: 265
+cantemo:*:265:
```
Comment (william-gr) [inline GIDs]: I dont like this diverging from FreeBSD, someone may take it meanwhile and make our lives harder.

Perhaps lets try miwi acquire a gid/uid upstream? Meanwhile we could pick a number in the middle to decrease the chance for someone to take it.

File: net/iconik_ui/Makefile [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# $FreeBSD$
+
+PORTNAME=	iconik_ui
+PORTVERSION=	1
+CATEGORIES=	net
+
+MAINTAINER=	waqar@ixsystems.com
+COMMENT=	Web Interface for iconik
+
+USES=	python:3.6
```
Comment (william-gr) [inline net/iconik_ui/Makefile]: Why not just `python`

File: GIDs (hunk 2 above)
Comment (sonicaj) [reply GIDs]: @miwi-fbsd can you please see if we can acquire a gid/uid upstream ? For now i'll proceed with a number in the middle then

File: net/iconik/distinfo (hunk 1 above)
Comment (sonicaj) [reply net/iconik/distinfo]: I'll email Joe to see if we can make that happen

**Pull request merged**